.github/workflows/validate-new-issue.yml

```diff
@@ -9,7 +9,22 @@ jobs:
     - uses: actions/checkout@v2
     - name: "Validate issue against templates"
       shell: bash
+      env:
+        GITHUB_TOKEN: ${{ github.token }}
       run: |
+
+        # Trust users who belong to the getsentry org.
+        check_membership() {
+          gh api "https://api.github.com/orgs/getsentry/members/${{ github.actor }}" 2> /dev/null
+          return 0  # prevent GHA from exiting on non-zero return code from the previous line
+        }
+        if [ -z "$(check_membership)" ]; then
```

Comment:
```suggestion
        # Trust users who belong to the getsentry org.
        if [ gh api "https://api.github.com/orgs/getsentry/members/${{ github.actor }}" 2> /dev/null ]; then
```

Reply:
I guess we _can_ use error code.

```
$ gh api "https://api.github.com/orgs/getsentry/members/chadwhitacrejjj"
{
gh: Not Found (HTTP 404)
  "message": "Not Found",
  "documentation_url": "https://docs.github.com/rest/reference/orgs#check-organization-membership-for-a-user"
}
$ echo $?
1
```
$ gh api "https://api.github.com/orgs/getsntry/members/chadwhitacre" 
{
  "message": "Not Found",
gh: Not Found (HTTP 404)
  "documentation_url": "https://docs.github.com/rest/reference/orgs#check-organization-membership-for-a-user"
}
$ echo $?                                                           
1
$ gh api "https://api.github.com/orgs/getsentry/members/chadwhitacre"   
$ echo $?                                                            
0
$
```